test-projects/social-login/server.js

```diff
@@ -0,0 +1,89 @@
+const keystone = require('@keystone-alpha/core');
+const { endAuthedSession } = require('@keystone-alpha/session');
+const {
+  facebookAuthEnabled,
+  githubAuthEnabled,
+  twitterAuthEnabled,
+  googleAuthEnabled,
+  wpAuthEnabled,
+  port,
+  staticRoute,
+  staticPath,
+} = require('./config');
+const { configureFacebookAuth } = require('./facebook');
+const { configureGitHubAuth } = require('./github');
+const { configureTwitterAuth } = require('./twitter');
+const { configureGoogleAuth } = require('./google');
+const { configureWPAuth } = require('./wordpress');
+
+const initialData = require('./data');
+
+keystone
+  .prepare({ port })
+  .then(async ({ server, keystone: keystoneApp }) => {
+    if (facebookAuthEnabled) {
+      configureFacebookAuth(keystoneApp, server);
+    }
+
+    if (githubAuthEnabled) {
+      configureGitHubAuth(keystoneApp, server);
+    }
+
+    if (twitterAuthEnabled) {
+      configureTwitterAuth(keystoneApp, server);
+    }
+
+    if (googleAuthEnabled) {
+      configureGoogleAuth(keystoneApp, server);
+    }
+
+    if (wpAuthEnabled) {
+      configureWPAuth(keystoneApp, server);
+    }
+
+    await keystoneApp.connect();
+
```

Comment:
`setupAuthRoutes()` should be called here.

Reply:
moved all call to `configure*()` after `keystoneApp.connect()`,
`setupAuthRoutes()` is per service, so it has to be called via `configure*()`